ios/PolyPodApp/PolyPod/SettingsView.swift

```diff
@@ -107,10 +108,18 @@ private struct MainSection: View {
                     label: "settings_licenses_title",
                     action: { activeSection = .licenses }
                 )
+                if !RuntimeInfo.isProduction {
+                    SettingsButton(label: "Export Logs",
```

Comment:
Missing i18n

Reply:
What is the process of adding a translation? I can add it myself? I can use Google translate for some German :D

Reply:
In this case, I think it's the best, but eventually all translations are done in Confluence and added through specific PRs. But for the time being, whatever you do is fine.